views/templates/hook/checkout.tpl

```diff
@@ -0,0 +1,122 @@
+{*
+ * This file is part of the official Pagantis module for PrestaShop.
+ *
+ * @author    Pagantis <integrations@pagantis.com>
+ * @copyright 2015-2016 Pagantis
+ * @license   proprietary
+ *}
+{if $pagantisIsEnabled}
+<div class="row">
+    <div class="col-xs-12">
+        <p class="payment_module Pagantis ps_version_{$ps_version|escape:'quotes'}">
+            <a class="pagantis-checkout ps_version_{$ps_version|escape:'quotes'} locale_{$locale|escape:'quotes'}" href="{$paymentUrl|escape:'html'}" title="{$pagantisTitle|escape:'quotes'}">
+                {$pagantisTitle|escape:'quotes'}
+                <span class="PagantisSimulator ps_version_{$ps_version|escape:'quotes'}"></span>
+            </a>
+        </p>
+        <script type="text/javascript">
+            function checkSimulatorContent() {
+                var pgContainer = document.getElementsByClassName("PagantisSimulator");
+                if(typeof pgContainer != 'undefined' && typeof pgContainer[0] != 'undefined') {
```

Comment:
This line could be replaced by: "pgContainer.length > 0"

Reply:
Changed